src/trace.rs

```diff
@@ -11,17 +11,32 @@ use tracing_subscriber::{layer::SubscriberExt, FmtSubscriber};
 pub use tracing_futures::Instrument;
 pub use tracing_tower::{InstrumentableService, InstrumentedService};
 
-pub fn init(color: bool, levels: &str, metrics: Option<metrics::Sink>) {
-    let subscriber = FmtSubscriber::builder()
-        .with_ansi(color)
-        .with_env_filter(levels)
-        .finish()
-        .with(Limit::default());
+pub fn init(color: bool, json: bool, levels: &str, metrics: Option<metrics::Sink>) {
+    let dispatch = if json {
+        let subscriber = FmtSubscriber::builder()
+            .with_ansi(color)
```

Comment:
Is there a use case for adding color to JSON logs? I thought they would normally be consumed only by other code.

Reply:
I believe this case most likely gets ignored but I'll remove this :+1: